### PR TITLE
Written assignment tasks should reference a FreeResponse level

### PR DESCRIPTION
**Pull request by joshlory**
Splitting this change up into multiple smaller PRs.  This change adds UI to choose a `FreeResponse` level for each `Plc::WrittenAssignmentTask`.  That level will be rendered for the user to complete.

### CODE REVIEW

File: dashboard/app/models/plc/enrollment_unit_assignment.rb [hunk 1]
```diff
@@ -45,9 +45,11 @@ def enroll_user_in_unit_with_learning_modules(learning_modules)
 
         module_assignment = Plc::EnrollmentModuleAssignment.find_or_create_by(plc_enrollment_unit_assignment: self, plc_learning_module: learning_module)
         learning_module.plc_tasks.each do |task|
-          Plc::EnrollmentTaskAssignment.find_or_create_by(plc_enrollment_module_assignment: module_assignment,
-                                                          plc_task: task, status: :not_started,
-                                                          type: task.class.task_assignment_type.name)
+          Plc::EnrollmentTaskAssignment.find_or_create_by(
+            plc_enrollment_module_assignment: module_assignment,
+            plc_task: task,
+            status: :not_started
```
Comment (mehalshah) [inline dashboard/app/models/plc/enrollment_unit_assignment.rb]: Use Plc::EnrollmentTaskAssignment::NOT_STARTED

File: dashboard/app/views/plc/tasks/edit/_written_assignment_task.html.haml [hunk 2]
```diff
@@ -4,8 +4,8 @@
 =form_for(@task, url: plc_task_path(@task)) do |f|
   .field
     = render partial: 'plc/tasks/edit/common_editable', locals: {f: f}
-    = f.label :assignment_description
-    = f.text_area :assignment_description, rows: 10, style: 'width: 100%', value: @task.assignment_description
+    = f.label :level_id
+    = f.select :level_id, options_for_select(FreeResponse.all.pluck(:name, :id), selected: @task.level_id)
```
Comment (mehalshah) [inline dashboard/app/views/plc/tasks/edit/_written_assignment_task.html.haml:8]: This is a fairly large select box, with 272 items in it. In practice, how usable is it? If it's hard to use, can you remove levels that are associated with non PD scripts?

File: dashboard/app/models/plc/written_enrollment_task_assignment.rb [hunk 3]
```diff
@@ -1,44 +0,0 @@
-# == Schema Information
```
Comment (mehalshah) [inline dashboard/app/models/plc/written_enrollment_task_assignment.rb:1]: If this is going away, then the "REVIEW_REJECTED" state needs to exist in enrollment_task_assignment. Unless we are modeling this differently

File: dashboard/app/views/plc/tasks/edit/_written_assignment_task.html.haml (hunk 2 above)
Comment (joshlory) [inline dashboard/app/views/plc/tasks/edit/_written_assignment_task.html.haml:8]: This edit experience will be temporary until we generate these objects from `.script` files.

File: dashboard/app/models/plc/written_enrollment_task_assignment.rb (hunk 3 above)
Comment (joshlory) [inline dashboard/app/models/plc/written_enrollment_task_assignment.rb:1]: My understanding is that the review status will be tracked by the underlying level, but I can add a note in Trello to track.

**Comment by mehalshah** [comment]
Rest LGTM

**Pull request merged**